src/ui/components/listClients/FilterBar.jsx

```diff
@@ -0,0 +1,55 @@
+import useStyles from './FilterBar.styles';
+import ChecksComponent from './ChecksComponent';
+import PriceComponent from './PriceComponent';
+import FilterIcon from './img/filter-icon.png';
+import Material from './Material';
+
+const FilterBar = () => {
+   const data = {
```

Comment:
put this data in a mock file you can create a new folder   inside de src, called mockedData, please set the data there